app/controllers/idv/phone_question_controller.rb

```diff
@@ -0,0 +1,35 @@
+module Idv
+  class PhoneQuestionController < ApplicationController
+    include ActionView::Helpers::DateHelper
+    include IdvStepConcern
+    include StepIndicatorConcern
+
+    before_action :confirm_verify_info_step_needed
```

Comment:
I had to think about why this is here and in HybridHandoffController. I think it's because we delete pii_from_doc from the session after the VerifyInfo step, so we need to make sure they're not past that step when they land here.

Reply:
yep ... after verify info accessing pii  is ☹️  [LG-10603](https://cm-jira.usa.gov/browse/LG-10603)